app/screens/user_profile/user_profile.js

```diff
@@ -197,6 +200,36 @@ export default class UserProfile extends PureComponent {
         return null;
     };
 
+    buildOrganizationBlock = () => {
+        const {theme, remoteClusterInfo} = this.props;
+        if (!remoteClusterInfo) {
+            return null;
+        }
+        const style = createStyleSheet(theme);
+        return (
+            <View>
+                <FormattedText
+                    id='mobile.routes.user_profile.organization'
+                    defaultMessage='ORGANIZATION'
+                    style={style.header}
+                />
+                <View style={style.organizationDataContainer}>
+                    <ChannelIcon
+                        isActive={true}
+                        isArchived={false}
+                        isBot={false}
+                        isInfo={true}
+                        size={16}
+                        shared={true}
+                        theme={theme}
+                        type={General.OPEN_CHANNEL}
+                    />
+                    <Text style={style.text}>{remoteClusterInfo.display_name}</Text>
```

Comment:
Are we 100% sure we always get this info on the `remoteClusterInfo` object?

Reply:
These changes are not a part of this PR. Sorry for the confusion.